_benchmark/user-guide/concepts.md

```diff
@@ -92,22 +92,22 @@ The following example workload shows all of the essential elements needed to cre
 
 A workload usually includes the following elements:
 
-- [indices]({{site.url}}{{site.baseurl}}/benchmark/workloads/indices/): Defines the relevant indexes and index templates used for the workload.
+- [indexes]({{site.url}}{{site.baseurl}}/benchmark/workloads/indices/): Defines the relevant indexes and index templates used for the workload.
```

Comment:
Let's switch this back to `indices` for the element since the element is referring to a code element within the workload.

```suggestion
- [indices]({{site.url}}{{site.baseurl}}/benchmark/workloads/indices/): Defines the relevant indexes and index templates used for the workload.
```

Reply:
Reversed change!